### ISSUE TITLE
 Can't add authorized email addresses / same problem like in closed #172 

### ISSUE THREAD

**Issue by dr-boss** [mikejac/node-red-contrib-google-smarthome#178]
Same problem, Authorized emails list is cleared. after every deploy. With user/pass login Synchronization work only from google device to node-red. But any change of device from node red is not send to google
version 0.1.6

_Originally posted by @dr-boss in https://github.com/mikejac/node-red-contrib-google-smarthome/issues/172#issuecomment-907525875_

**Comment by ckhmer1** [comment]
Hi,
Have You tried all the stuff in the original post?
If it is not working, could You please post the same info? Without info, we cannot help You.

**Comment by brianmay** [comment]
A big problem fixing this is that after the problem is fixed it isn't possible to reproduce it any more, and the people most likely to encounter this problem are new users who are unfamiliar with the code. :-(

Having said that, I believe this is the code that is suppose to save the email addresses. Seems simple enough. But maybe this isn't working for some reason. If the problem isn't here, most likely to be somewhere in this file.

https://github.com/mikejac/node-red-contrib-google-smarthome/blob/master/google-smarthome.html#L401-L405

**Comment by Caprico85** [comment]
I would fix it if I could. But I was not able to reproduce it. Tried on two existing installs with existing and new management nodes. Also tried it with a completely new Node-RED install. But is was always working as expected.

So I think it is either related to the browser or to a special version of node.js, Node-RED and/or this package.

@ckhmer1 also made a possible fix in 1e2f0d8e3dd9baf67e2a5922657385efc94fe49d. We used this to migrate the email addresses from older formats to the new JS-enhanced email list. But probalby it migrated older addresses once too often, overwriting new addresses.
